files/lmod

```diff
@@ -1,4 +1,4 @@
-LMOD_GENERAL (?:- lmod::)\s*username=%{USERNAME_PAT}, cluster=%{DATA:cluster}, arch=%{DATA:architecture}, jobid=%{DATA:jobid}
+LMOD_GENERAL (?:- lmod::)\s*username=%{USERNAME_PAT}, cluster=%{DATA:cluster}, jobid=%{DATA:jobid}
 LMOD_USERLOAD (?:yes|no)
 LMOD_LOAD userload=%{LMOD_USERLOAD:userload}, module=%{DATA:module_name}/%{DATA:module_version}, fn=%{GREEDYDATA:module_path}
 LMOD_CMD cmd=%{DATA:command}, args=%{GREEDYDATA:arguments}
```

Comment:
is there a way to tell logstash that `arguments` is a space-separated *list* of strings?